src/command/cmd_funcs.c

```diff
@@ -4952,6 +4950,8 @@ cmd_sendfile(ProfWin* window, const char* const command, gchar** args)
         goto out;
     }
 
+    FILE* fh = fdopen(fd, "rb");
```

Comment:
```suggestion
    auto_FILE FILE* fh = fdopen(fd, "rb");
```

and remove the potential `fclose()` calls

Reply:
This one is passed to HTTPUpload though. And we already free more of that stuff later on. So I think its best not to mix it to free some stuff and not others.

Reply:
yeah, right I missed that.